### PR TITLE
Add MIX level 2 skeleton

### PR DESCRIPTION
**Pull request by estark37**

### CODE REVIEW

**Comment by estark37** [comment]
@mikewest PTAL?

File: level2.src.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  TODO. Call the autoupgrading algorithm from Fetch.
+
+  TODO. Something about blocking insecure downloads from secure contexts (integrate with HTML).
+</section>
+
+<section>
+  <h2 id="obsolescences">Obsolescences</h2>
+
+  TODO. block-all-mixed-content is obsolete. Clarify the value of
+  upgrade-insecure-requests (upgrades blockable content as well as
```
Comment (mikewest) [inline level2.src.html]: I was assuming we'd deprecate UIR too?

Comment (estark37) [reply level2.src.html]: Discussed a bit this morning -- depends on whether we want to autoupgrade blockable resources or just leave them as they are.

**Comment by estark37** [comment]
Thanks @mikewest! Can you go ahead and merge this? Don't think I have permission.

**Comment by mikewest** [comment]
> Thanks @mikewest! Can you go ahead and merge this? Don't think I have permission.

Hey, look! Now you do!

**Pull request merged**